.gitignore

```diff
@@ -19,7 +19,7 @@
 /.htaccess
 /web.config
 /.php_cs.cache
-/plugins/media-action/smartcrop/focus.json
+/plugins/media-action/focus/focus.json
```

Comment:
As we store data in focus directory we can do something like 

`/plugins/media-action/focus/*`

to ensure that any file would be deleted, so in future developments it may help

Reply:
Actually, it's can't be ```/plugins/media-action/focus/*``` as this the main focus plugin folder. We need to create a new folder like ```/plugins/media-action/focus/data/``` and then store ```focus.json``` there after that we can neglect it.

Reply:
ah, got it, can you create this `data` folder? it seems better to keep data seperately

Reply:
Done, Added ```data``` folder